mono/mini/driver.c

```diff
@@ -1568,6 +1568,27 @@ apply_root_domain_configuration_file_bindings (MonoDomain *domain, char *root_do
 
 }
 
+static int
+mono_enable_interp (const char *opts)
+{
+#ifndef DISABLE_INTERPRETER
+	mono_use_interpreter = TRUE;
+	if (opts)
+		mono_interp_parse_options (opts);
+#else
+	g_printerr ("Mono Warning: --interpreter not enabled in this runtime.\n");
```

Comment:
What does this mean? We print an error with the warning message which makes little sense to me.

Could we do
```
#ifdef DISABLE_INTERPRETER
    g_warning ("Mono IL interpreter support is missing.\n");
#endif
```

instead?

Reply:
we do have the same kind of messages for e.g. llvm as well. `g_printerr` just means it prints to `stderr` (or whatever is available on that system; it's better than using `fprintf(stderr, ...)` imho)

Reply:
Using #ifdef you could avoid the #else

Further, `Mono Warning: --interpreter not enabled in this runtime` sentence is quite confusing to me.

Reply:
okay, updated. I hope it is more clear now.